### PR TITLE
promisify and lipstickify

### PR DESCRIPTION
**Pull request by SamVerschueren**
Just touched the bare minimum in `cli.js` to let the tests pass. Not sure about all the doc changes in the readme as well. If you want it back like it was, let me know and I'll revert!

Feedback welcome.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 os: osx
 language: node_js
 node_js:
+  - '6'
   - '5'
```
Comment (sindresorhus) [inline .travis.yml]: Drop `5`

**Comment by sindresorhus** [comment]
Yeah, just drop the CLI. There are better CLI's for this.

File: readme.md [hunk 2]
```diff
@@ -28,34 +28,45 @@ displayNotification({
...
 
+Type: `string`
+
+Title of the notification.
+
 ##### subtitle
 
+Type: `string`
+
+Subtitle of the nofication.
```
Comment (sindresorhus) [inline readme.md]: typo

File: readme.md [hunk 3]
```diff
@@ -28,34 +28,45 @@ displayNotification({
 
 ## API
 
-### displayNotification(options, callback)
-
-Everything is a string and optional, except you need to supply at least a `title` or `text`.
```
Comment (sindresorhus) [inline readme.md]: This shouldn't be removed

**Comment by SamVerschueren** [comment]
Dropped the cli and fixed all the rest

**Comment by sindresorhus** [comment]
Looks good 👌

**Comment by SamVerschueren** [comment]
Sweet!

**Pull request merged**